src/Form/input-classifiers.js

```diff
@@ -40,7 +40,10 @@ const getRelatedText = (el, form) => {
 
     // If the container has a select element, remove its contents to avoid noise
     const noisyText = container.querySelector('select')?.textContent || ''
-    return removeExcessWhitespace(container.textContent?.replace(noisyText, ''))
+    const sanitizedText = removeExcessWhitespace(container.textContent?.replace(noisyText, ''))
+    // If the text is longer than 50 chars it's too noisy and likely to yield false positives, so return ''
+    if (sanitizedText.length < 50) return sanitizedText
```

Comment:
This is a pretty arbitrary constant, can you put it in the constants file or similar?

Reply:
Good idea. Done! 👍